pkg/api/admin_users.go

```diff
@@ -70,14 +70,14 @@ func (hs *HTTPServer) AdminCreateUser(c *contextmodel.ReqContext) response.Respo
 	usr, err := hs.userService.Create(c.Req.Context(), &cmd)
 	if err != nil {
 		if errors.Is(err, org.ErrOrgNotFound) {
-			return response.Error(400, err.Error(), nil)
+			return response.Error(http.StatusBadRequest, err.Error(), nil)
```

Comment:
@sakjur any idea how to better handle this? I would prefer to use [response.Err()](https://github.com/grafana/grafana/blob/9765c98b3dfb1f7a615ec78f409bc5ec4b0954cc/pkg/api/response/response.go#L255) but in this specific case we want to return a different status code (400 Bad Request) than the actual error code (404 Not Found)

Reply:
oh, that's a good one, I'm not sure. You could wrap it in another layer of errutil.Error, but that's a bit hacky